.eslintrc.json

```diff
@@ -39,6 +39,7 @@
     "func-call-spacing": ["error", "never"],
     "prefer-arrow-callback": "error",
     "no-trailing-spaces": "error",
-    "comma-dangle": ["error", "never"]
+    "comma-dangle": ["error", "never"],
+    "no-console": "off"
```

Comment:
@sebastiankliem  @cmfcmf 
Actually we already got a logger library called Winston. You can see how to use it in index.js: https://github.com/frederike-ramin/gamification/blob/0857295d73aaaaf275ec036fe0f6e22bf53bbfac/src/index.js#L12

So please remove this rule and use Winston instead of console.